src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/PassengerSelectionSheet.tsx

```diff
@@ -0,0 +1,82 @@
+import {BottomSheetContainer} from '@atb/components/bottom-sheet';
+import {ScreenHeaderWithoutNavigation} from '@atb/components/screen-header';
+import {PurchaseOverviewTexts, useTranslation} from '@atb/translations';
+import {ScrollView} from 'react-native';
+import React from 'react';
+import {StyleSheet} from '@atb/theme';
+import {FullScreenFooter} from '@atb/components/screen-footer';
+import {Button} from '@atb/components/button';
+import {Confirm} from '@atb/assets/svg/mono-icons/actions';
+import {MultipleTravellersSelection} from './Travellers/MultipleTravellersSelection';
+import {SingleTravellerSelection} from './Travellers/SingleTravellerSelection';
+import {
+  UserProfileWithCount,
+  useUserCountState,
+} from './Travellers/use-user-count-state';
+
+type PassengerSelectionSheetProps = {
+  selectionMode: 'multiple' | 'single';
```

Comment:
Can this type be `TravellerSelectionMode`?

Reply:
Yes